docs/secure-boot.md

```diff
@@ -1,183 +1,180 @@
-## Secure boot and firmware verification requirements
+# Secure boot and firmware verification requirements
 
-Copyright © 2018 Arm Limited.
```

Comment:
Please add this back and update it.
```suggestion
Copyright © 2018-2019 Arm Limited.
```

Reply:
Fixed in https://github.com/ARMmbed/meta-mbl/pull/637/commits/e25ca8a425be2a0dd065aef99369924edb291d02.